### DISCUSSION TITLE
oData Date Format support /Date(...)/

### DISCUSSION THREAD

**Discussion by dschoeneborn** [grafana/grafana-json-datasource#297] [Q&A] [answered]
Hello,

first of all I have to say that I am very impressed about the DataSource. 
However, there is one PainPoint currently. I am trying to access an oData service. This returns with TimeStamps in the following format:

`/Date(836524800000)/`

Is it currently already possible to display this format correctly?

Example: https://services.odata.org/v2/northwind/northwind.svc/Orders?$orderby=OrderDate&$format=JSON

Regards
Dennis

**Comment by kokfai83** [comment, marked as answer]
I think you can use Override in the dashboard, standard options -> unit -> Datetime ISO.  then it show 2021-08-11 07:59:59

**Reply by dschoeneborn** [reply]
Works very well, thank you!
However, I can't calculate any new columns from it. For example, calculating a duration between two timestamps is not possible because the source data types are strings.
Do you have a solution for this as well?
